app/src/sharedTest/java/org/oppia/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -34,40 +36,59 @@ class InputInteractionViewTestActivityTest {
     }
   }
 
+  /* ktlint-disable max-line-length */
   @Test
   fun testFractionInputInteractionView_withInputtedNegativeWholeNumberText_hasCorrectPendingAnswer() {
-    val activityScenario = ActivityScenario.launch(InputInteractionViewTestActivity::class.java)
+    val activityScenario = ActivityScenario.launch(
+      InputInteractionViewTestActivity::class.java
+    )
     onView(withId(R.id.test_fraction_input_interaction_view)).perform(typeText("-9"))
     activityScenario.onActivity { activity ->
       val pendingAnswer = activity.fractionInteractionViewModel.getPendingAnswer()
       assertThat(pendingAnswer.answer).isInstanceOf(InteractionObject::class.java)
-      assertThat(pendingAnswer.answer.objectTypeCase).isEqualTo(InteractionObject.ObjectTypeCase.FRACTION)
+      assertThat(pendingAnswer.answer.objectTypeCase).isEqualTo(
+        InteractionObject.ObjectTypeCase.FRACTION
+      )
       assertThat(pendingAnswer.answer.fraction.isNegative).isEqualTo(true)
       assertThat(pendingAnswer.answer.fraction.wholeNumber).isEqualTo(9)
     }
   }
+  /* ktlint-enable max-line-length */
```

Comment:
Can we move this below the function  name itself instead of below the entire block?  (Also I think you can do `// ktlint-disable max-line-length` next to the function name itself and it should disable it just for that line, instead of disabling and then enabling in a block , have you tried that?

Reply:
Done